Dalamud/Utility/Util.cs

```diff
@@ -36,6 +37,8 @@ public static class Util
 
     private static ulong moduleStartAddr;
     private static ulong moduleEndAddr;
+    
+    public static int copyButtonIndex;
```

Comment:
this shouldn't be public api - can we make it private and maybe have an `internal static` method to reset it on `DataWindow.Draw()`?

Reply:
Removed with next pr update.

Solution: Have the buttons be unique based on the contents of what would be copied to clipboard.
If there are duplicate buttons, they will still trigger the copy correctly.